src/index.js

```diff
@@ -28,12 +28,16 @@ export default class Voices {
   }
 
   byLangCode (langCode) {
-    this.voices = this.voices.filter(voice => voice.LanguageCode.toLowerCase() === langCode.toLowerCase())
+    this.voices = this.voices.filter(
+      voice => voice.LanguageCode.toLowerCase() === langCode.toLowerCase()
+    )
     return this
   }
 
   byGender (gender) {
-    this.voices = this.voices.filter(voice => voice.Gender.toLowerCase() === gender.toLowerCase())
+    this.voices = this.voices.filter(
+      voice => voice.Gender.toLowerCase() === gender.toLowerCase()
+    )
```

Comment:
could you commit only the changes related to your feature, please?

Reply:
Where is the conflict?

Reply:
Try to merge master inside your branch and you see the conflicts. There are three files with conflicts to be resolved before the merge.

![image](https://user-images.githubusercontent.com/2037349/68550774-5922f700-0406-11ea-86a0-f9cd7b4d1883.png)